<!-- --> 
<launch>

	<node name="control_drone_node" pkg="direction_beta" type="control_direction_node" output="screen">
		      <param name="Kp" value="0.25"/>
		      <param name="Kd" value="0.25"/>
		      <param name="velocity_limit" value="0.2"/>
	   	      <param name="new_yaw" value="0.0"/>
		      <param name="new_altitude" value="0.5"/>
		      <rosparam>new_position:  [0.0, 0.0]</rosparam>  <!-- [X, Y] -->
		      <rosparam>virtual_fence:  [3.0, -2.45, 1.85, -2.4]</rosparam>  <!-- [maxX, minX, maxY, minY] -->
	</node>

</launch>
